netdissect.h

```diff
@@ -221,7 +221,8 @@ struct netdissect_options {
   const char *ndo_protocol;	/* protocol */
   jmp_buf ndo_truncated;	/* jmp_buf for setjmp()/longjmp() */
   void *ndo_last_mem_p;		/* pointer to the last allocated memory chunk */
-  int ndo_packet_number;	/* print a packet number in the beginning of line */
+  int ndo_packet_number;	/* flag: print a packet number in the beginning of line */
+  int ndo_packets_captured;     /* current packet number */
```

Comment:
"current packet number" could be read as "current number of packets" or "number of current packet"; in a sense, the two are the same (if tcpdump has captured N packets, the current packet is the Nth), but "current count of captured packets" might be a little clearer.

Reply:
Changed the comment in a new pull request.